sources/packages/api/src/services/form/form.service.ts

```diff
@@ -74,12 +74,11 @@ export class FormService {
     try {
       const authHeader = await this.createAuthHeader();
       const submissionResponse = await axios.post(
-        `${this.config.formsUrl}/${formName}/submission?dryrun=1`,
+        `${this.config.formsUrl}/${formName}/submission?dryRun=1`,
```

Comment:
just to check, did the query parameter `dryRun` actually changed?

Reply:
Nope, I just thought that it would be easy to read and also get rid of the spell check waning.